docs.html

```diff
@@ -373,7 +373,165 @@ <h2 class="list-topic">Components</h2>
 </code>
 </section>
 
-      </div>
+<section class="section" >
+  <div class="component-title" >Card</div>
+  <div class="component-detail" >
+
+    Cards are surfaces that display content and actions on a single topic.
+    They should be easy to scan for relevant and actionable information. Elements, like text and images, should be placed on them in a way that clearly indicates hierarchy.
+  </div>
+
+<div class="component-subhead" >Basic Card</div>
+
+
+<div class="card-container">
+  <img
+    src="https://picsum.photos/380/240"
+    alt="nature img"
+    class="random-img"
+    loading="eager"
```

Comment:
the `loading="lazy"`  attribute should be used for the images which are not in the viewport at initial loading, it improves the performance of the website.

Reply:
thank you ,  doing the changes